test/jdk/java/net/httpclient/ForbiddenHeadTest.java

```diff
@@ -381,7 +381,7 @@ public void setup() throws Exception {
     public void teardown() throws Exception {
         authClient = noAuthClient = null;
         Thread.sleep(100);
-        AssertionError fail = TRACKER.check(500);
+        AssertionError fail = TRACKER.check(1500);
```

Comment:
We have other tests where we use a higher timeout like this. So I think it's OK to increase this value to try and prevent this failure.